internal/pkg/agent/install/install_windows.go

```diff
@@ -80,6 +82,12 @@ func withServiceOptions(username string, groupName string) ([]serviceOpt, error)
 // gives user the ability to control the service, needed when installed with --unprivileged or
 // ReExec is not possible on Windows.
 func servicePostInstall(ownership utils.FileOwner) error {
+	// Modify registry to allow logging to eventlog as "Elastic Agent".
+	err := eventlog.InstallAsEventCreate(paths.ServiceName, eventlog.Info|eventlog.Warning|eventlog.Error)
+	if err != nil && !strings.Contains(err.Error(), "registry key already exists") {
```

Comment:
Does this work for both `elastic-agent install` and `elastic-agent install --unprivileged`?

Reply:
I assume CI will tell us this since it happens unconditionally and CI tests unprivileged agents on Windows.

Reply:
`--unprivileged` should have no effect.  This code only runs during install, where you still need Admin privs, a little farther down in this function we set the SecurityDescriptor, also something that needs Admin privs.